ui/src/overview.tsx

```diff
@@ -81,7 +82,14 @@ function AddCriteriaButton(props: { group: string | GroupKind }) {
       groupId = props.group;
       dispatch(insertCriteria({ cohortId: cohort.id, groupId, criteria }));
     } else {
-      const action = dispatch(insertGroup(cohort.id, props.group, criteria));
+      const action = dispatch(
+        insertGroup(
+          cohort.id,
+          "Group" + String(cohort.groups.length + 1),
```

Comment:
not sure whether this is a good way. my concern from 2 sides. one is Do you think this is long-- to create a variable. But this is only used once. The other hand is seems like a little hard code?

Reply:
The AoU UI is a bit more subtle than this. It leaves the names blank at creation, and displays "Group X" as a fallback in that case. What that accomplishes is that when you have two groups and delete "Group 1", an unnamed "Group 2" will instead revert to displaying "Group 1". If you set the name at creation time then "Group 2" will stay the same and adding a new group will add a second "Group 2". We should use the same behavior.

As for the second question, what would you propose instead of creating the name like this? Unless we had an internationalization setup to make it work in different languages I'm not sure where would be a better place to put it.